arsen.musheghyan/repo/Lesson_3/exercise_05/exercise_05.py

```diff
@@ -0,0 +1,4 @@
+print ("Exercise N 5")
```

Comment:
RE: Do not use additional prints in the code. Follow the exercise requirements. You can use comments instead.